src/ray/core_worker/core_worker.cc

```diff
@@ -16,11 +16,16 @@ CoreWorker::CoreWorker(
       object_interface_(worker_context_, raylet_client_, store_socket) {
   int rpc_server_port = 0;
   if (worker_type_ == WorkerType::WORKER) {
-    RAY_CHECK(execution_callback != nullptr);
-    task_execution_interface_ = std::unique_ptr<CoreWorkerTaskExecutionInterface>(
-        new CoreWorkerTaskExecutionInterface(worker_context_, raylet_client_,
+    // TODO(edoakes): assert that execution_callback isn't nullptr once Python core
+    // worker task execution is implemented.
```

Comment:
can we check `execution_callback != nullptr || language == PYTHON` instead?

Reply:
Great idea!